### ISSUE TITLE
Misleading outcome when using "stop-domain" without specifying domain

### ISSUE THREAD

**Issue by ggam** [payara/Payara#1841]
# Description #
----------

The `stop-domain` command always defaults to the `domain1` domain. But when you've started another domain (e.g. `payaradomain`), calling `stop-domain` gives you the following output:

```
c:\payarabin>asadmin stop-domain
CLI306: Warning - The server located at C:\servers\payara-4.1.2.173-nightly\glassfish\domains\domain1 is not running.
Command stop-domain executed successfully.
```

It's the "command executed successfully" what's confusing to me. The command has finished correctly, but doing nothing. 

## Expected Outcome

I think a message like the following would be clearer:
```
c:\payarabin>asadmin stop-domain
Trying to stop domain1...
CLI306: Warning - The server located at C:\servers\payara-4.1.2.173-nightly\glassfish\domains\domain1 is not running.
domain1 could not be stopped. You may want to specify a different domain to stop.  Running domains: payaradomain.

Command stop-domain executed successfully.
```

Another possibility would be to both default to `domain1` and then `payaradomain` if `domain1` was not started.

## Environment ##

- **Payara Version**: 4.1.2.173 nightly
- **Edition**: Full
- **JDK Version**: Oracle JDK 8
- **Operating System**:  Windows 10

**Comment by Cousjava** [comment]
Created internal issue PAYARA-2113 to change it

**Comment by smillidge** [comment]
Fixed by #2169
